FreeRTOS/corePKCS11
# Update mbedtls to v3.4.0 and v2.28.3

Update mbedtls to v3.4.0 and v2.28.3

Description
-----------
Update to MbedTLS v3.4.0 and v2.28.3 when building and running tests.
    
Adjust mbedtls_configure.sh script accordingly.

Test Steps
-----------
Ran unit tests and integration / system tests

Checklist:
----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
